tsdb/cursors/arrayvalues.gen.go.tmpl

```diff
@@ -0,0 +1,259 @@
+package cursors
```

Comment:
This tmpl was removed in 2b175291be69bc3f750ed74161d2a2e093dcbef9, when the file it generates was moved to a different module. The generated content is still the same so it looks like we haven't updated the output file since the move happened.